plugins/out_stackdriver/stackdriver.c

```diff
@@ -696,6 +696,38 @@ static int get_severity_level(severity_t * s, const msgpack_object * o,
     return -1;
 }
 
+static int get_stream(msgpack_object_map map)
+{
+    int i;
+    int len_stream;
+    int val_size;
+    msgpack_object k;
+    msgpack_object v;
+
+    len_stream = sizeof(STDOUT) - 1;
+    for (i = 0; i < map.size; i++) {
+        k = map.ptr[i].key;
+        v = map.ptr[i].val;
+        if (k.type == MSGPACK_OBJECT_STR &&
+            strncmp(k.via.str.ptr, "stream", k.via.str.size) == 0) {
+            val_size = v.via.str.size;
+            if (val_size == len_stream) {
+                if (strncmp(v.via.str.ptr, STDOUT, val_size) == 0) {
+                    return STREAM_STDOUT;
+                }
+                else if (strncmp(v.via.str.ptr, STDERR, val_size) == 0) {
+                    return STREAM_STDERR;
+                }
+                else {
+                    return STREAM_UNKNOWN;
+                }
```

Comment:
Actually we don't need this `else` block. 

I would prefer move this out (outside of `for` even) - if we don't know the stream, return `STREAM_UNKNOWN` in any condition.

Reply:
updated.